Mathlib/RepresentationTheory/Tannaka.lean

```diff
@@ -117,3 +117,27 @@ lemma equivHom_inj [Nontrivial k] [DecidableEq G] : Function.Injective (equivHom
 end FiniteGroup
 
 end TannakaDuality
+
+end
+
+/-- Let `k` be an integral domain and `G` an arbitrary finite set.
+Then any algebra morphism `φ : (G → k) →ₐ[k] k` is an evaluation map. -/
+lemma eval_of_algHom {k G : Type*} [CommRing k] [IsDomain k] [DecidableEq G] [Fintype G]
+    (φ : (G → k) →ₐ[k] k) : ∃ (s : G), φ = Pi.evalAlgHom _ _ s := by
```

Comment:
The statement doesn't need Finite, so we should use [Finite G] instead of [Fintype G].